views/js/qtiCreator/widgets/interactions/mediaInteraction/states/Question.js

```diff
@@ -352,17 +396,26 @@ define([
          * Can be called again, as needed
          */
         const renderForm = function renderForm() {
+            const sequenceRepeats = parseInt(interaction.attr('data-sequence-repeats') || 1, 10) || 1;
+            const hidePlayer = !!interaction.hasClass('hide-player');
+            const maxPlays = parseInt(interaction.attr('maxPlays') || 0, 10);
+            const uiMaxPlays = Math.round(maxPlays / sequenceRepeats);
```

Comment:
Trying to reconcile `repeats` & `maxPlays`.
Maybe we should disable `maxPlays` if `repeats` exist.
Or maybe when user changes `repeats` from 3 to 4, we should automatically change `maxPlay`s from 3 to 4 in the UI.

Reply:
I think that when the player is hidden (which it always should be if sequential - even if we didn't enforce it yet), the maxPlays concept no longer applies. I think `repeats` is not necessarily involved. I would be happy enough if we made `maxPlays` 0 and disabled when `sequential` is true.